Index: src/AeroTaxi/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package AeroTaxi;\r\n\r\nimport javax.swing.*;\r\n//hola mundo\r\n\r\npublic class Main {\r\n\r\n    public static void main(String[] args) {\r\n\r\n        String manaGay;\r\n        //// es cierto\r\n\r\n\r\n\r\n        double toga;\r\n        int seba = 007;\r\n        int trolo;\r\n        float gato;\r\n\r\n        try {\r\n            UIManager.setLookAndFeel(\"javax.swing.plaf.nimbus.NimbusLookAndFeel\");\r\n        } catch (Exception e1) {\r\n            e1.printStackTrace();\r\n        }\r\n\r\n        SwingUtilities.invokeLater(new Runnable() {\r\n            @Override\r\n            public void run() {\r\n                MainWindow mainwindows = new MainWindow();\r\n            }\r\n        });\r\n    }\r\n}\r\n\r\n// seba trolo
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/AeroTaxi/Main.java	(revision 3cb3b089a2029b47b1e7ca3e96d5880d434071a7)
+++ src/AeroTaxi/Main.java	(date 1590957781222)
@@ -7,15 +7,7 @@
 
     public static void main(String[] args) {
 
-        String manaGay;
-        //// es cierto
-
 
-
-        double toga;
-        int seba = 007;
-        int trolo;
-        float gato;
 
         try {
             UIManager.setLookAndFeel("javax.swing.plaf.nimbus.NimbusLookAndFeel");
Index: src/AeroTaxi/Guille.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package AeroTaxi;\r\n\r\nimport javax.swing.*;\r\n\r\npublic class Guille {\r\n    private JComboBox comboBox1;\r\n    private JPanel panel1;\r\n    private JList list1;\r\n    private int guille = 69;\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/AeroTaxi/Guille.java	(revision 3cb3b089a2029b47b1e7ca3e96d5880d434071a7)
+++ src/AeroTaxi/Guille.java	(date 1590957675378)
@@ -6,5 +6,5 @@
     private JComboBox comboBox1;
     private JPanel panel1;
     private JList list1;
-    private int guille = 69;
+
 }
