Index: src/AeroTaxi/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package AeroTaxi;\r\n\r\nimport javax.swing.*;\r\n//hola mundo\r\n\r\npublic class Main {\r\n\r\n    public static void main(String[] args) {\r\n\r\n        String manaGay;\r\n        //// es cierto\r\n\r\n\r\n\r\n        int seba = 007;\r\n        float gato;\r\n\r\n        try {\r\n            UIManager.setLookAndFeel(\"javax.swing.plaf.nimbus.NimbusLookAndFeel\");\r\n        } catch (Exception e1) {\r\n            e1.printStackTrace();\r\n        }\r\n\r\n        SwingUtilities.invokeLater(new Runnable() {\r\n            @Override\r\n            public void run() {\r\n                MainWindow mainwindows = new MainWindow();\r\n            }\r\n        });\r\n    }\r\n}\r\n\r\n// seba trolo
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/AeroTaxi/Main.java	(revision 01d538f278e4816c6c9c181d7afac1b466dca5bb)
+++ src/AeroTaxi/Main.java	(date 1590958553384)
@@ -7,9 +7,6 @@
 
     public static void main(String[] args) {
 
-        String manaGay;
-        //// es cierto
-
 
 
         int seba = 007;
Index: src/AeroTaxi/Guille.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package AeroTaxi;\r\n\r\nimport javax.swing.*;\r\n\r\npublic class Guille {\r\n    private JComboBox comboBox1;\r\n    private JPanel panel1;\r\n    private JList list1;\r\n    private int guille = 69;\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/AeroTaxi/Guille.java	(revision 01d538f278e4816c6c9c181d7afac1b466dca5bb)
+++ src/AeroTaxi/Guille.java	(date 1590958553408)
@@ -6,5 +6,5 @@
     private JComboBox comboBox1;
     private JPanel panel1;
     private JList list1;
-    private int guille = 69;
+
 }
